.github/workflows/build-crowdin-translation-packages.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Crowdin Translation Packages
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 6 * * *"
+
+jobs:
+  build-crowdin-translation-packages:
+    defaults:
+      run:
+        shell: pwsh
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Orchard CMS
+        env:
+          CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY_ORCHARD_CMS }}
+        run: (Invoke-WebRequest -Uri 'https://api.crowdin.com/api/project/orchard-cms/export?key=${{ env.CROWDIN_API_KEY }}').Content
```

Comment:
The v1 API is deprecated: https://developer.crowdin.com/assets/api/api_v1_docs_crowdin.pdf Perhaps this is the v2 equivalent? https://developer.crowdin.com/api/v2/#operation/api.projects.translations.exports.post